packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -83,15 +83,15 @@ export interface BaseServiceOptions {
    *
    * Valid values are: PropagateTagFromType.SERVICE, PropagateTagFromType.TASK_DEFINITION or PropagateTagFromType.NONE
    *
-   * @default - PropagatedTagSource.SERVICE if EC2 or Fargate Service, otherwise PropagatedTagSource.NONE.
+   * @default - PropagatedTagSource.NONE.
```

Comment:
nit: can we make this `@default PropagatedTagSource.NONE` ?

Reply:
Updated